syfoalerts.yaml

```diff
@@ -11,15 +11,22 @@ spec:
       #prependText: '<!here> | '
   alerts:
     - alert: syfo-app-nede
-      expr: up{app=~"pdf-gen|syfoarbeidsgivertilgang|syfogsak|syfoinntektsmelding|syfonarmesteleder|syfoservicestrangler|syfosoknadvarsel|syfosoknad|syfosyketilfelle|syfoaltinn|syfovarsel",job="kubernetes-pods"} == 0
+      expr: up{app=~"syfoarbeidsgivertilgang|syfonarmesteleder|syfoservicestrangler|syfosoknad|syfosyketilfelle",job="kubernetes-pods"} == 0
       for: 2m
```

Comment:
Burde denne være litt høyere? ~10 min kanskje? Feil som fikser seg selv i løpet av 4-7 min er kanskje ikke verd å angripe?

Reply:
Dette er snakk om synkrone apper..? Er det ikke dumt om de er nede i 10 minutter uten at vi vet det..?

Reply:
Ja det er vel egentlig det. 2min er kanskje nok til å unngå falske positiver ved flytting av pods osv

Reply:
Hvis de er veldig trege til å starte kan vi nok øke den litt. Tenker at disse alertene er litt prøving og feiling. Er to min for lite øker vi bare :)